controller.js

```diff
@@ -0,0 +1,111 @@
+
+const Controller = () => {
+    const moudl = moudle()
+    const vie = view()
+    const addingnewpost = (post) => {
```

Comment:
for arrow functions, if it has only one parameter you do not need to add `()`
also, you should be camelCasing you veriables in JS
`const addingNewPost = post => {`

Reply:
ok thanks .